.travis.yml

```diff
@@ -30,6 +30,7 @@ env:
   global:
     - ANSIBLE_CMD="ansible-playbook -i environments/local -e docker_image_prefix=testing"
     - GRADLE_PROJS_SKIP=""
+    - secure: "ChSOkhFvCMEj7dvthS9QUpfNDl8/x0lOTm2IHu2pupBRRGSJxCEZ6T4cIl6CAHWid1sSqMJjBBwZRPGzZsBDTWdWWaUBTJMmL4fRtW4MjP7P9Oi2Zjml7QFoehgMB18jKYmgTNSFTA9nE7pbOWW3aN6ootLvIB5jpfdgU07z81xIr5eXTOOjrYU2hFi5nRSiuNsK2HjogCUN+iIB09ubBYd/XJ6OCUQraH0BMj0FdMlyFRvYvrpddoCGmrKb6Axt2QCTRwfht8FTOX1CRBy/PLIJj3k4GtuOXnrSpFIdKREyGfqqiiPIYnncyNSFp8kzDA0N4/WfY7dzkm3TBoF2dox54V3pQcvnpJH9afAw8Jj7gxfTWaKhP/LKV+Fu1u3mKiWLpVmiqB6otpybl5KQnHGMxBgurvVxncOC/X34rjj32IGU+NAR7lO5u5Wn2FUeJw1sNe8HynPOdHE4SHhlttfv1LYD2BG4IJ5Qo2N6SFDI6OPIAWHyLiPwkg/vR1BOFxFxZBYkYliO1BZToFbZC7FFNntwgF1Cr2Ty+yuAXhNB//azXa4YEJ/VwVgizr3uhOyE0iwlRdogQnQhhYacSS/c4VLcpyX/8e52OC4JI5ZWUXMoqC1vcd+H6wz8Kx/qwY5brVmSNERIGhHAr5paUqBYEa5HsMACgtMYx75qOj4="
```

Comment:
would be nice to have a comment saying what this secure entry is used for so it could later be removed if it becomes obsolete.